### PR TITLE
Add tests for the ErrorPage view

### PR DESCRIPTION
**Pull request by j4ckofalltrades**
Refs #124

### CODE REVIEW

File: client/src/views/ErrorPage.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
 .center-container.fadeIn(v-wow data-wow-duration="1s")
   h1 oops!
   .quote-container.sentence
-    | An error occured.
```
Comment (j4ckofalltrades) [inline client/src/views/ErrorPage.vue:5]: Fix typo.

Comment (jayehernandez) [reply client/src/views/ErrorPage.vue:5]: thanks 😅

**Review by jayehernandez — APPROVED** [review]
Thank you for contributing to Letra ☀️, @j4ckofalltrades!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Comment by jayehernandez** [comment]
@all-contributors please add @j4ckofalltrades for testing

**Pull request merged**